DAQ/src/ArtFragmentsFromDTCEvents_module.cc

```diff
@@ -164,7 +164,8 @@ void art::ArtFragmentsFromDTCEvents::produce(Event& event) {
     }
 
     if (makeCRVFrag_ > 0) { // CRV
-      auto crvSEvents = bb.getSubsystemData(DTCLib::DTC_Subsystem::DTC_Subsystem_CRV);
+//      auto crvSEvents = bb.getSubsystemData(DTCLib::DTC_Subsystem::DTC_Subsystem_CRV);
+      auto crvSEvents = bb.getSubsystemData(DTCLib::DTC_Subsystem::DTC_Subsystem_Tracker);  //currently wrongly encoded in the DTC Subevent header
```

Comment:
Is this a patch to correct the wrongly-encoded header, or is this a comment that the decoding doesn't work?

Reply:
Currently, the DTC SubEvent headers don't seem to have the correct source IDs encoded. It's set to 0 (tracker) instead of 2 (CRV). I think this is the case for all sub systems, and solutions are currently being discussed.